src/graphics/gfx_print.cpp

```diff
@@ -44,11 +47,25 @@ int SuperTextPixLen(int SuperN, const char* SuperChars, int Font)
     default:
     case 1:
     case 4:
-        len = SuperN * 18;
+    {
+        //len = SuperN * 18;
+        for(int i = 0; i < SuperN; i++)
+        {
+            len += 18;
```

Comment:
Strictly, this should depend on the font override, yes?

Reply:
It's an OLD font engine's size measurer. However, because of a mistake, I hadn't added the forwarding into the new font engine. I did it just now.